frontend/src/pages/CourseSelector/courseMenu/CourseMenu.jsx

```diff
@@ -174,25 +178,43 @@ export default function CourseMenu() {
   );
 }
 
-const MenuItem = ({ selected, courseCode, activeCourse, setActiveCourse }) => {
+const MenuItem = ({ selected, courseCode, activeCourse, setActiveCourse, accurate }) => {
   const dispatch = useDispatch();
   const handleClick = () => {
     dispatch(courseTabActions("ADD_TAB", courseCode));
     setActiveCourse(courseCode);
   };
 
+  const renderAccurateNote = () => {
+    if (!accurate){
+      return (<WarningIcon text="This course info may be inaccurate" />);
+    }
+  }
+
   return (
     <Menu.Item
       className={`text menuItemText ${selected !== undefined && "bold"} 
       ${activeCourse === courseCode && "activeCourse"}`}
       key={courseCode}
       onClick={handleClick}
     >
-      {courseCode}
+      {courseCode} {renderAccurateNote()}
     </Menu.Item>
   );
 };
 
+const WarningIcon = ({ text }) => {
+  return (
+    <Tooltip placement="top" title={text}>
+      <IoWarningOutline
```

Comment:
Ok, so this is a bit subjective but I kind of like the TiWarning icon better since it has soft edges. 
![image](https://user-images.githubusercontent.com/68191852/155881126-0eb7f2aa-8a76-41fe-9d25-eed0f91f2fa8.png)
What do you think?